boards/PDM/Inc/Io/Io_Efuse.h

```diff
@@ -0,0 +1,257 @@
+/**
+ * @brief Driver for interfacing with the 22XS4200 E-Fuse via SPI
+ * @Settings Efuse max SPI transfer rate: 8MHz
+ *           SPI Clock Polarity: 0 (SCK low-level idle state)
+ *           SPI Clock Phase: 2nd edge (slave samples MOSI on SCK falling edge)
+ *           Slave Select: Active Low (Must be toggled between SPI messages)
+ */
+#pragma once
+
+#include <stdint.h>
+#include <stdbool.h>
+#include "main.h"
+#include "App_SharedExitCode.h"
+
+struct Efuse;
+
+// Enumeration for the status types
+enum Efuse_Status
+{
+    StatusType_NoFault = (0 << 0), // No fault
```

Comment:
Should we capitalize these?

Reply:
No, you're right I'll change this since doesn't follow our coding standard. Enum style was taken form here:
https://github.com/madcowswe/ODrive/blob/306000f46e5d04f21177ee05fcd11469e5282295/Firmware/Drivers/DRV8301/drv8301.h#L216
for reference